datafusion/sql/tests/sql_integration.rs

```diff
@@ -4631,6 +4631,8 @@ fn roundtrip_statement() -> Result<()> {
             "select (id-1)/2, count(*) / (sum(id/10)-1) as agg_expr from (select (id-1) as id from person) group by id",
             "select CAST(id/2 as VARCHAR) NOT LIKE 'foo*' from person where NOT EXISTS (select ta.j1_id, tb.j2_string from j1 ta join j2 tb on (ta.j1_id = tb.j2_id))",
             r#"select "First Name" from person_quoted_cols"#,
+            "select DISTINCT id FROM person;",
+            "select DISTINCT on (id) id, first_name from person order by id;",
```

Comment:
Can you also please add a test like this (without an order by)?

```sql
select DISTINCT on (id) id, first_name from person;
```

Reply:
Added, thanks for your review. :)